### ISSUE TITLE
`Unable to start debugging session` when trying to debug tests

### ISSUE THREAD

**Issue by ChristoWolf** [dotnet/vscode-csharp#6934]
## Issue Description ##

Hi!

Whenever I try to debug tests, the `C# Dev Kit - Test Explorer` reports `Unable to start debugging session` and the targeted tests in the TESTING view stay yellow and are never executed.
Stopping them and trying to run tests does nothing then.
Only restarting VS Code resets the TESTING view.

Why am I reporting this here instead of at [vscode-dotnettools](https://github.com/microsoft/vscode-dotnettools)?
Thing is that this only happens with v2.18.16 (and newer) of the C# extension, but v2.15.30 works fine.
Changing the version of the C# Dev Kit did not change anything.

Seems to be unrelated to [this](https://github.com/microsoft/vscode-dotnettools/issues/910).

## Steps to Reproduce ##

1. With a similar environment, attempt to debug a test via `Debug Test`.
2. Navigate to *OUTPUT* and select *C# Dev Kit - Test Explorer* in the drop-down there.

## Expected Behavior ##

`Debugger attached` and the debugger is actually attached to the test run.

## Actual Behavior ##

```
Created Test Controller
Using vstest from dotnet sdk in [C:\Program Files\dotnet\sdk\8.0.100].
Initialized Test Explorer Server [15632]
Test Store Folder: c:\Users\<redacted>\AppData\Roaming\Code\User\workspaceStorage\7fa6afdb1ce7a6d4b881651aeb6cf65b\ms-dotnettools.csdevkit
Test data store opened in 2.345 sec.
Initialized project system provider.
<redacted> projects added, 0 changed, 0 removed.
Scheduling discovery: <redacted> [1709017421856]
========== Starting test discovery ==========
No test is available in <redacted>. Make sure that test discoverer & executors are registered and platform & framework version settings are appropriate and try again.
[MSTest][Discovery][<redacted>] UTA001: TestClass attribute defined on non-public class <redacted>
========== Test discovery finished: <redacted> Tests found in 48.9 sec ==========
<redacted> tests discovered from <redacted>
========== Starting test run ==========
Unable to start debugging session.
```

![image](https://github.com/dotnet/vscode-csharp/assets/52244643/ee3590ff-826d-476f-9e65-95cbeb1226d2)

### C# log ###

<details>
<summary>Expand...</summary>
<code>[LanguageServerHost] [01:42:13.868][Start]workspace/buildOnlyDiagnosticIds
[LanguageServerHost] [01:42:13.895][End]workspace/buildOnlyDiagnosticIds
[LanguageServerHost] [01:42:20.958][Start]workspace/buildOnlyDiagnosticIds
[LanguageServerHost] [01:42:20.981][End]workspace/buildOnlyDiagnosticIds
[LanguageServerHost] [01:42:21.372][Start]workspace/buildOnlyDiagnosticIds
[LanguageServerHost] [01:42:21.389][End]workspace/buildOnlyDiagnosticIds
[LanguageServerHost] [01:42:22.467][Start]workspace/buildOnlyDiagnosticIds
[LanguageServerHost] [01:42:22.484][End]workspace/buildOnlyDiagnosticIds
[LanguageServerHost] [01:42:22.558][Start]workspace/buildOnlyDiagnosticIds
[LanguageServerHost] [01:42:22.577][End]workspace/buildOnlyDiagnosticIds
[LanguageServerHost] [01:42:22.671][Start]workspace/buildOnlyDiagnosticIds
[LanguageServerHost] [01:42:22.677][Start]workspace/buildOnlyDiagnosticIds
[LanguageServerHost] [01:42:22.690][End]workspace/buildOnlyDiagnosticIds
[LanguageServerHost] [01:42:22.698][End]workspace/buildOnlyDiagnosticIds
[LanguageServerHost] [01:42:23.192][Start]workspace/buildOnlyDiagnosticIds
[LanguageServerHost] [01:42:23.218][End]workspace/buildOnlyDiagnosticIds
</code>
</details>

### C# LSP Trace Logs ###

<details>
<summary>Expand...</summary>
<code>[Trace - 2:43:10 PM] Sending request 'workspace/buildOnlyDiagnosticIds - (205)'.
[Trace - 2:43:10 PM] Received notification 'window/logMessage'.
Params: {
    "type": 4,
    "message": "[LanguageServerHost] [01:43:10.157][Start]workspace/buildOnlyDiagnosticIds"
}

[Trace - 2:43:10 PM] Received response 'workspace/buildOnlyDiagnosticIds - (205)' in 23ms.
Result: {
    "ids": [
        "CS1607",
        "CS0169",
        "CS0414",
        "CS0067",
        "CS0649",
        "CS0656",
        "CS0518",
        "CS5001",
        "CS0028",
        "CS0017",
        "CS7022",
        "CS1556",
        "CS0402",
        "CS1558",
        "CS1555",
        "CS8892",
        "CS0148",
        "CS8078",
        "CS7038",
        "CS0204",
        "CS0570",
        "CS8004",
        "CS8006",
        "CS8005",
        "CS8008",
        "CS4007",
        "CS8178",
        "CS4013",
        "CS1969",
        "CS9026",
        "CS9068",
        "CS9144",
        "CS9148",
        "CS9149",
        "CS9153",
        "CS9154",
        "CS9155",
        "CS9156",
        "CS9158",
        "CS9159",
        "CS9160",
        "CS9163",
        "CS9177",
        "CS9178",
        "CS9207",
        "CS8419",
        "CS8420",
        "CA1016",
        "CA1014",
        "CA1017",
        "CA1017",
        "CA1724",
        "CA1724",
        "CA1501",
        "CA1502",
        "CA1505",
        "CA1506",
        "CA1509",
        "CA1852",
        "CA5360",
        "CA5399",
        "CA5400",
        "CA5380",
        "CA5381",
        "CA5367",
        "CA5401",
        "CA5402",
        "CA2301",
        "CA2302",
        "CA2321",
        "CA2322",
        "CA2329",
        "CA2330",
        "CA2311",
        "CA2312",
        "CA2327",
        "CA2328",
        "CA5387",
        "CA5388",
        "CA5362",
        "CA5396",
        "CA5391",
        "CA5395",
        "CA5382",
        "CA5383",
        "CA1812",
        "CA1824",
        "RS1010",
        "RS1011",
        "RS1016",
        "RS1019",
        "RS2002",
        "RS2003",
        "RS2005",
        "RS2006",
        "RS2007",
        "RS2007",
        "RS2007",
        "RS2007",
        "RS1022",
        "RS1022",
        "RS1010",
        "RS1011",
        "RS1016",
        "RS1019",
        "RS2002",
        "RS2003",
        "RS2005",
        "RS2006",
        "RS2007",
        "RS2007",
        "RS2007",
        "RS2007",
        "RS1022",
        "RS1022"
    ]
}

[Trace - 2:43:10 PM] Received notification 'window/logMessage'.
Params: {
    "type": 4,
    "message": "[LanguageServerHost] [01:43:10.178][End]workspace/buildOnlyDiagnosticIds"
}

[Trace - 2:43:23 PM] Sending request 'workspace/buildOnlyDiagnosticIds - (206)'.
[Trace - 2:43:23 PM] Received notification 'window/logMessage'.
Params: {
    "type": 4,
    "message": "[LanguageServerHost] [01:43:23.351][Start]workspace/buildOnlyDiagnosticIds"
}

[Trace - 2:43:23 PM] Received response 'workspace/buildOnlyDiagnosticIds - (206)' in 19ms.
Result: {
    "ids": [
        "CS1607",
        "CS0169",
        "CS0414",
        "CS0067",
        "CS0649",
        "CS0656",
        "CS0518",
        "CS5001",
        "CS0028",
        "CS0017",
        "CS7022",
        "CS1556",
        "CS0402",
        "CS1558",
        "CS1555",
        "CS8892",
        "CS0148",
        "CS8078",
        "CS7038",
        "CS0204",
        "CS0570",
        "CS8004",
        "CS8006",
        "CS8005",
        "CS8008",
        "CS4007",
        "CS8178",
        "CS4013",
        "CS1969",
        "CS9026",
        "CS9068",
        "CS9144",
        "CS9148",
        "CS9149",
        "CS9153",
        "CS9154",
        "CS9155",
        "CS9156",
        "CS9158",
        "CS9159",
        "CS9160",
        "CS9163",
        "CS9177",
        "CS9178",
        "CS9207",
        "CS8419",
        "CS8420",
        "CA1016",
        "CA1014",
        "CA1017",
        "CA1017",
        "CA1724",
        "CA1724",
        "CA1501",
        "CA1502",
        "CA1505",
        "CA1506",
        "CA1509",
        "CA1852",
        "CA5360",
        "CA5399",
        "CA5400",
        "CA5380",
        "CA5381",
        "CA5367",
        "CA5401",
        "CA5402",
        "CA2301",
        "CA2302",
        "CA2321",
        "CA2322",
        "CA2329",
        "CA2330",
        "CA2311",
        "CA2312",
        "CA2327",
        "CA2328",
        "CA5387",
        "CA5388",
        "CA5362",
        "CA5396",
        "CA5391",
        "CA5395",
        "CA5382",
        "CA5383",
        "CA1812",
        "CA1824",
        "RS1010",
        "RS1011",
        "RS1016",
        "RS1019",
        "RS2002",
        "RS2003",
        "RS2005",
        "RS2006",
        "RS2007",
        "RS2007",
        "RS2007",
        "RS2007",
        "RS1022",
        "RS1022",
        "RS1010",
        "RS1011",
        "RS1016",
        "RS1019",
        "RS2002",
        "RS2003",
        "RS2005",
        "RS2006",
        "RS2007",
        "RS2007",
        "RS2007",
        "RS2007",
        "RS1022",
        "RS1022"
    ]
}

[Trace - 2:43:23 PM] Received notification 'window/logMessage'.
Params: {
    "type": 4,
    "message": "[LanguageServerHost] [01:43:23.369][End]workspace/buildOnlyDiagnosticIds"
}

[Trace - 2:43:28 PM] Sending request 'workspace/buildOnlyDiagnosticIds - (207)'.
[Trace - 2:43:28 PM] Received notification 'window/logMessage'.
Params: {
    "type": 4,
    "message": "[LanguageServerHost] [01:43:28.281][Start]workspace/buildOnlyDiagnosticIds"
}

[Trace - 2:43:28 PM] Received response 'workspace/buildOnlyDiagnosticIds - (207)' in 23ms.
Result: {
    "ids": [
        "CS1607",
        "CS0169",
        "CS0414",
        "CS0067",
        "CS0649",
        "CS0656",
        "CS0518",
        "CS5001",
        "CS0028",
        "CS0017",
        "CS7022",
        "CS1556",
        "CS0402",
        "CS1558",
        "CS1555",
        "CS8892",
        "CS0148",
        "CS8078",
        "CS7038",
        "CS0204",
        "CS0570",
        "CS8004",
        "CS8006",
        "CS8005",
        "CS8008",
        "CS4007",
        "CS8178",
        "CS4013",
        "CS1969",
        "CS9026",
        "CS9068",
        "CS9144",
        "CS9148",
        "CS9149",
        "CS9153",
        "CS9154",
        "CS9155",
        "CS9156",
        "CS9158",
        "CS9159",
        "CS9160",
        "CS9163",
        "CS9177",
        "CS9178",
        "CS9207",
        "CS8419",
        "CS8420",
        "CA1016",
        "CA1014",
        "CA1017",
        "CA1017",
        "CA1724",
        "CA1724",
        "CA1501",
        "CA1502",
        "CA1505",
        "CA1506",
        "CA1509",
        "CA1852",
        "CA5360",
        "CA5399",
        "CA5400",
        "CA5380",
        "CA5381",
        "CA5367",
        "CA5401",
        "CA5402",
        "CA2301",
        "CA2302",
        "CA2321",
        "CA2322",
        "CA2329",
        "CA2330",
        "CA2311",
        "CA2312",
        "CA2327",
        "CA2328",
        "CA5387",
        "CA5388",
        "CA5362",
        "CA5396",
        "CA5391",
        "CA5395",
        "CA5382",
        "CA5383",
        "CA1812",
        "CA1824",
        "RS1010",
        "RS1011",
        "RS1016",
        "RS1019",
        "RS2002",
        "RS2003",
        "RS2005",
        "RS2006",
        "RS2007",
        "RS2007",
        "RS2007",
        "RS2007",
        "RS1022",
        "RS1022",
        "RS1010",
        "RS1011",
        "RS1016",
        "RS1019",
        "RS2002",
        "RS2003",
        "RS2005",
        "RS2006",
        "RS2007",
        "RS2007",
        "RS2007",
        "RS2007",
        "RS1022",
        "RS1022"
    ]
}

[Trace - 2:43:28 PM] Received notification 'window/logMessage'.
Params: {
    "type": 4,
    "message": "[LanguageServerHost] [01:43:28.302][End]workspace/buildOnlyDiagnosticIds"
}

[Trace - 2:43:33 PM] Sending request 'workspace/buildOnlyDiagnosticIds - (208)'.
[Trace - 2:43:33 PM] Received notification 'window/logMessage'.
Params: {
    "type": 4,
    "message": "[LanguageServerHost] [01:43:33.442][Start]workspace/buildOnlyDiagnosticIds"
}

[Trace - 2:43:33 PM] Received response 'workspace/buildOnlyDiagnosticIds - (208)' in 63ms.
Result: {
    "ids": [
        "CS1607",
        "CS0169",
        "CS0414",
        "CS0067",
        "CS0649",
        "CS0656",
        "CS0518",
        "CS5001",
        "CS0028",
        "CS0017",
        "CS7022",
        "CS1556",
        "CS0402",
        "CS1558",
        "CS1555",
        "CS8892",
        "CS0148",
        "CS8078",
        "CS7038",
        "CS0204",
        "CS0570",
        "CS8004",
        "CS8006",
        "CS8005",
        "CS8008",
        "CS4007",
        "CS8178",
        "CS4013",
        "CS1969",
        "CS9026",
        "CS9068",
        "CS9144",
        "CS9148",
        "CS9149",
        "CS9153",
        "CS9154",
        "CS9155",
        "CS9156",
        "CS9158",
        "CS9159",
        "CS9160",
        "CS9163",
        "CS9177",
        "CS9178",
        "CS9207",
        "CS8419",
        "CS8420",
        "CA1016",
        "CA1014",
        "CA1017",
        "CA1017",
        "CA1724",
        "CA1724",
        "CA1501",
        "CA1502",
        "CA1505",
        "CA1506",
        "CA1509",
        "CA1852",
        "CA5360",
        "CA5399",
        "CA5400",
        "CA5380",
        "CA5381",
        "CA5367",
        "CA5401",
        "CA5402",
        "CA2301",
        "CA2302",
        "CA2321",
        "CA2322",
        "CA2329",
        "CA2330",
        "CA2311",
        "CA2312",
        "CA2327",
        "CA2328",
        "CA5387",
        "CA5388",
        "CA5362",
        "CA5396",
        "CA5391",
        "CA5395",
        "CA5382",
        "CA5383",
        "CA1812",
        "CA1824",
        "RS1010",
        "RS1011",
        "RS1016",
        "RS1019",
        "RS2002",
        "RS2003",
        "RS2005",
        "RS2006",
        "RS2007",
        "RS2007",
        "RS2007",
        "RS2007",
        "RS1022",
        "RS1022",
        "RS1010",
        "RS1011",
        "RS1016",
        "RS1019",
        "RS2002",
        "RS2003",
        "RS2005",
        "RS2006",
        "RS2007",
        "RS2007",
        "RS2007",
        "RS2007",
        "RS1022",
        "RS1022"
    ]
}

[Trace - 2:43:33 PM] Received notification 'window/logMessage'.
Params: {
    "type": 4,
    "message": "[LanguageServerHost] [01:43:33.463][End]workspace/buildOnlyDiagnosticIds"
}
</code>
</details>

## Environment information ##

**VSCode version**: 1.86.2
**C# Extension**: 2.18.16
**Using OmniSharp**: false
**C# Dev Kit**: v1.4.6 (pre-release) (but happens also with older versions)

<details><summary>Dotnet Information</summary>
.NET SDK:
 Version:           8.0.100
 Commit:            57efcf1350
 Workload version:  8.0.100-manifests.1eabc556

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.19045
 OS Platform: Windows
 RID:         win-x64
 Base Path:   C:\Program Files\dotnet\sdk\8.0.100\

.NET workloads installed:
 Workload version: 8.0.100-manifests.1eabc556
 [ios]
   Installation Source: VS 17.8.34309.116
   Manifest Version:    17.0.8478/8.0.100
   Manifest Path:       C:\Program Files\dotnet\sdk-manifests\8.0.100\microsoft.net.sdk.ios\17.0.8478\WorkloadManifest.json
   Install Type:              Msi

 [android]
   Installation Source: VS 17.8.34309.116
   Manifest Version:    34.0.43/8.0.100
   Manifest Path:       C:\Program Files\dotnet\sdk-manifests\8.0.100\microsoft.net.sdk.android\34.0.43\WorkloadManifest.json
   Install Type:              Msi

 [maui-windows]
   Installation Source: VS 17.8.34309.116
   Manifest Version:    8.0.3/8.0.100
   Manifest Path:       C:\Program Files\dotnet\sdk-manifests\8.0.100\microsoft.net.sdk.maui\8.0.3\WorkloadManifest.json
   Install Type:              Msi

 [maccatalyst]
   Installation Source: VS 17.8.34309.116
   Manifest Version:    17.0.8478/8.0.100
   Manifest Path:       C:\Program Files\dotnet\sdk-manifests\8.0.100\microsoft.net.sdk.maccatalyst\17.0.8478\WorkloadManifest.json
   Install Type:              Msi

Host:
  Version:      8.0.0
  Architecture: x64
  Commit:       5535e31a71

.NET SDKs installed:
  5.0.408 [C:\Program Files\dotnet\sdk]
  6.0.317 [C:\Program Files\dotnet\sdk]
  6.0.407 [C:\Program Files\dotnet\sdk]
  7.0.405 [C:\Program Files\dotnet\sdk]
  8.0.100-rc.2.23502.2 [C:\Program Files\dotnet\sdk]
  8.0.100 [C:\Program Files\dotnet\sdk]

.NET runtimes installed:
  Microsoft.AspNetCore.App 3.1.32 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 5.0.17 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 6.0.15 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 6.0.22 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 6.0.25 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 7.0.14 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 7.0.15 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 8.0.0-rc.2.23480.2 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 8.0.0 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 3.1.32 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 5.0.17 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 6.0.15 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 6.0.22 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 6.0.25 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 7.0.5 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 7.0.14 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 7.0.15 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 8.0.0-rc.2.23479.6 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 8.0.0 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.WindowsDesktop.App 3.1.32 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 5.0.17 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 6.0.15 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 6.0.22 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 6.0.25 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 7.0.5 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 7.0.14 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 7.0.15 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 8.0.0-rc.2.23479.10 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 8.0.0 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]

Other architectures found:
  x86   [C:\Program Files (x86)\dotnet]
    registered at [HKLM\SOFTWARE\dotnet\Setup\InstalledVersions\x86\InstallLocation]

Environment variables:
  Not set

global.json file:
  Not found

Learn more:
  https://aka.ms/dotnet/info

Download .NET:
  https://aka.ms/dotnet/download
</details>
<details><summary>Visual Studio Code Extensions</summary>

|Extension|Author|Version|Folder Name|
|---|---|---|---|
|aldrico-s-gruvbox|HeavenAldrico|0.2.3|heavenaldrico.aldrico-s-gruvbox-0.2.3|
|apple-dark|Mac3|0.0.1|mac3.apple-dark-0.0.1|
|autodocstring|njpwerner|0.6.1|njpwerner.autodocstring-0.6.1|
|azure-account|ms-vscode|0.11.6|ms-vscode.azure-account-0.11.6|
|azure-pipelines|ms-azure-devops|1.228.0|ms-azure-devops.azure-pipelines-1.228.0|
|azure-repos|ms-vscode|0.38.0|ms-vscode.azure-repos-0.38.0|
|becave-theme|beeinger|1.0.1|beeinger.becave-theme-1.0.1|
|better-comments|aaron-bond|3.0.2|aaron-bond.better-comments-3.0.2|
|blackberry-theme|Gustavosta|1.0.0|gustavosta.blackberry-theme-1.0.0|
|Bookmarks|alefragnani|13.4.2|alefragnani.bookmarks-13.4.2|
|celestial|apvarun|0.3.0|apvarun.celestial-0.3.0|
|code-runner|formulahendry|0.12.1|formulahendry.code-runner-0.12.1|
|code-spell-checker|streetsidesoftware|3.0.1|streetsidesoftware.code-spell-checker-3.0.1|
|copilot|GitHub|1.166.0|github.copilot-1.166.0|
|copilot-chat|GitHub|0.12.2|github.copilot-chat-0.12.2|
|csdevkit|ms-dotnettools|1.4.6|ms-dotnettools.csdevkit-1.4.6-win32-x64|
|csharp|ms-dotnettools|2.18.16|ms-dotnettools.csharp-2.18.16-win32-x64|
|csharp-format-usings|gaoshan0621|0.0.4|gaoshan0621.csharp-format-usings-0.0.4|
|dark-horizon|mcagampan|0.4.2|mcagampan.dark-horizon-0.4.2|
|dart-code|Dart-Code|3.82.0|dart-code.dart-code-3.82.0|
|debugpy|ms-python|2024.0.0|ms-python.debugpy-2024.0.0-win32-x64|
|EditorConfig|EditorConfig|0.16.4|editorconfig.editorconfig-0.16.4|
|errorlens|usernamehw|3.16.0|usernamehw.errorlens-3.16.0|
|flutter|Dart-Code|3.82.0|dart-code.flutter-3.82.0|
|git-rename|ambooth|1.3.0|ambooth.git-rename-1.3.0|
|gitlens|eamodio|14.8.2|eamodio.gitlens-14.8.2|
|go|golang|0.41.1|golang.go-0.41.1|
|gostmortem-table|AOS|0.0.1|aos.gostmortem-table-0.0.1|
|gruvbox-material-icon-theme|JonathanHarty|1.1.5|jonathanharty.gruvbox-material-icon-theme-1.1.5|
|hcl|hashicorp|0.3.2|hashicorp.hcl-0.3.2|
|insert-unicode|brunnerh|0.15.1|brunnerh.insert-unicode-0.15.1|
|intellicode-api-usage-examples|VisualStudioExptTeam|0.2.8|visualstudioexptteam.intellicode-api-usage-examples-0.2.8|
|isort|ms-python|2023.10.1|ms-python.isort-2023.10.1|
|json|ZainChen|2.0.2|zainchen.json-2.0.2|
|jupyter|ms-toolsai|2024.1.1|ms-toolsai.jupyter-2024.1.1-win32-x64|
|jupyter-keymap|ms-toolsai|1.1.2|ms-toolsai.jupyter-keymap-1.1.2|
|jupyter-renderers|ms-toolsai|1.0.17|ms-toolsai.jupyter-renderers-1.0.17|
|latex-workshop|James-Yu|9.18.2|james-yu.latex-workshop-9.18.2|
|MagicPython|magicstack|1.1.0|magicstack.magicpython-1.1.0|
|packer|4ops|0.3.0|4ops.packer-0.3.0|
|polacode|pnp|0.3.4|pnp.polacode-0.3.4|
|pop-n-lock-theme-vscode|Luxcium|3.32.188|luxcium.pop-n-lock-theme-vscode-3.32.188|
|powershell|ms-vscode|2024.0.0|ms-vscode.powershell-2024.0.0|
|prettify-json|mohsen1|0.0.3|mohsen1.prettify-json-0.0.3|
|python|ms-python|2024.0.1|ms-python.python-2024.0.1|
|python-environment-manager|donjayamanne|1.2.4|donjayamanne.python-environment-manager-1.2.4|
|python-extension-pack|donjayamanne|1.7.0|donjayamanne.python-extension-pack-1.7.0|
|quicktype|quicktype|12.0.46|quicktype.quicktype-12.0.46|
|rainbow-csv|mechatroner|3.11.0|mechatroner.rainbow-csv-3.11.0|
|regex|chrmarti|0.4.0|chrmarti.regex-0.4.0|
|remote-containers|ms-vscode-remote|0.338.1|ms-vscode-remote.remote-containers-0.338.1|
|remote-explorer|ms-vscode|0.4.1|ms-vscode.remote-explorer-0.4.1|
|remote-repositories|ms-vscode|0.40.0|ms-vscode.remote-repositories-0.40.0|
|remote-ssh|ms-vscode-remote|0.108.0|ms-vscode-remote.remote-ssh-0.108.0|
|remote-ssh-edit|ms-vscode-remote|0.86.0|ms-vscode-remote.remote-ssh-edit-0.86.0|
|remote-wsl|ms-vscode-remote|0.86.0|ms-vscode-remote.remote-wsl-0.86.0|
|remotehub|GitHub|0.62.0|github.remotehub-0.62.0|
|selected-lines-count|gurumukhi|1.4.0|gurumukhi.selected-lines-count-1.4.0|
|sequoia|wicked-labs|1.24.0|wicked-labs.sequoia-1.24.0|
|seven-dark|lianginet|0.1.5|lianginet.seven-dark-0.1.5|
|sqltools|mtxr|0.28.1|mtxr.sqltools-0.28.1|
|terraform|hashicorp|2.29.4|hashicorp.terraform-2.29.4-win32-x64|
|test-adapter-converter|ms-vscode|0.1.9|ms-vscode.test-adapter-converter-0.1.9|
|theme-verdandi|be5invis|2.0.1|be5invis.theme-verdandi-2.0.1|
|uuid-generator|netcorext|0.0.5|netcorext.uuid-generator-0.0.5|
|vagrant|bbenoist|0.5.0|bbenoist.vagrant-0.5.0|
|vsc-python-indent|KevinRose|1.18.0|kevinrose.vsc-python-indent-1.18.0|
|vscode-antlr4|mike-lischke|2.4.6|mike-lischke.vscode-antlr4-2.4.6|
|vscode-autohotkey-plus-plus|mark-wiemer|5.0.3|mark-wiemer.vscode-autohotkey-plus-plus-5.0.3|
|vscode-commons|redhat|0.0.6|redhat.vscode-commons-0.0.6|
|vscode-docker|ms-azuretools|1.29.0|ms-azuretools.vscode-docker-1.29.0|
|vscode-dotnet-runtime|ms-dotnettools|2.0.1|ms-dotnettools.vscode-dotnet-runtime-2.0.1|
|vscode-fileutils|sleistner|3.10.3|sleistner.vscode-fileutils-3.10.3|
|vscode-github-actions|github|0.26.2|github.vscode-github-actions-0.26.2|
|vscode-html-css|ecmel|2.0.9|ecmel.vscode-html-css-2.0.9|
|vscode-jupyter-cell-tags|ms-toolsai|0.1.8|ms-toolsai.vscode-jupyter-cell-tags-0.1.8|
|vscode-jupyter-slideshow|ms-toolsai|0.1.5|ms-toolsai.vscode-jupyter-slideshow-0.1.5|
|vscode-markdownlint|DavidAnson|0.54.0|davidanson.vscode-markdownlint-0.54.0|
|vscode-packer-powertools|szTheory|0.3.0|sztheory.vscode-packer-powertools-0.3.0|
|vscode-pipelines-theme|ryanraposo|0.0.3|ryanraposo.vscode-pipelines-theme-0.0.3|
|vscode-proto3|zxh404|0.5.5|zxh404.vscode-proto3-0.5.5|
|vscode-pylance|ms-python|2024.2.2|ms-python.vscode-pylance-2024.2.2|
|vscode-solution-explorer|fernandoescolar|0.8.5|fernandoescolar.vscode-solution-explorer-0.8.5|
|vscode-sort-json|richie5um2|1.20.0|richie5um2.vscode-sort-json-1.20.0|
|vscode-task|task|0.3.2|task.vscode-task-0.3.2|
|vscode-taskfile|paulvarache|0.0.5|paulvarache.vscode-taskfile-0.0.5|
|vscode-theme-gruvbox-minor|adamsome|2.7.2|adamsome.vscode-theme-gruvbox-minor-2.7.2|
|vscode-thunder-client|rangav|2.18.0|rangav.vscode-thunder-client-2.18.0|
|vscode-todo-highlight|jgclark|2.0.8|jgclark.vscode-todo-highlight-2.0.8|
|vscode-vault|owenfarrell|2.2.3|owenfarrell.vscode-vault-2.2.3|
|vscode-xml|redhat|0.26.1|redhat.vscode-xml-0.26.1-win32-x64|
|vscode-yaml|redhat|1.14.0|redhat.vscode-yaml-1.14.0|
|vscodeintellicode|VisualStudioExptTeam|1.2.30|visualstudioexptteam.vscodeintellicode-1.2.30|
|vscodeintellicode-csharp|ms-dotnettools|0.1.26|ms-dotnettools.vscodeintellicode-csharp-0.1.26-win32-x64|
|vsliveshare|ms-vsliveshare|1.0.5905|ms-vsliveshare.vsliveshare-1.0.5905|
|xml|DotJoshJohnson|2.5.1|dotjoshjohnson.xml-2.5.1|;

</details>

**Comment by gregg-miskelly** [comment]
Does it work if you use the 'Run Test' (instead of 'Debug Test') button in Test Explorer?
If the problem only happens with debugging, can you [enable debug protocol logging using the VS Code setting](https://github.com/dotnet/vscode-csharp/wiki/Enabling-C%23-debugger-logging#vs-code-settings) and grab the log from the debug console?

**Comment by ChristoWolf** [comment]
Hi @gregg-miskelly!

It works perfectly fine when using "Run Test".

Thanks for the hint with the debugger logging!
How heavily does this affect performance if enabled all the time?
Here is the output:
```
-> (C) {"command":"initialize","arguments":{"clientID":"vscode","clientName":"Visual Studio Code","adapterID":"coreclr","pathFormat":"path","linesStartAt1":true,"columnsStartAt1":true,"supportsVariableType":true,"supportsVariablePaging":true,"supportsRunInTerminalRequest":true,"locale":"en","supportsProgressReporting":true,"supportsInvalidatedEvent":true,"supportsMemoryReferences":true,"supportsArgsCanBeInterpretedByShell":true,"supportsMemoryEvent":true,"supportsStartDebuggingRequest":true},"type":"request","seq":1}
-> (C) {"command":"attach","arguments":{"justMyCode":true,"requireExactSource":false,"symbolOptions":{"searchMicrosoftSymbolServer":true,"searchNuGetOrgSymbolServer":true},"type":"coreclr","name":".NET Test Attach","request":"attach","processId":32136,"attachRequestId":"3c38aac4-c783-46fd-8eb5-47616d1b4c4a","hotReloadOnSave":true,"hotReloadVerbosity":"minimal","stopAtEntry":false,"enableStepFiltering":true,"logging":{"exceptions":true,"moduleLoad":true,"programOutput":true,"browserStdOut":true,"elapsedTiming":false,"threadExit":false,"processExit":true,"engineLogging":false,"diagnosticsLog":{"protocolMessages":false,"dispatcherMessages":"none","debugEngineAPITracing":"none","debugRuntimeEventTracing":false,"expressionEvaluationTracing":false,"startDebuggingTracing":false},"consoleUsageMessage":true},"suppressJITOptimizations":false,"allowFastEvaluate":true,"__sessionId":"fe127111-da9f-4966-80df-ac9d46035f20"},"type":"request","seq":2}
```

**Comment by gregg-miskelly** [comment]
Thanks! You are running into #6918. The debugger has a bug in its handling of the `searchNuGetOrgSymbolServer` option. As a work around until we put out the next version of the debugger, remove that option. If you need symbols from nuget.org, you can still download them by adding the server (`https://symbols.nuget.org/download/symbols`) to the search path.

**Comment by ChristoWolf** [comment]
Just tried it out, that seems to be the cause!
I will for now just use the old version until the one with the fix is released as I would like to keep my settings unchanged.

Thanks for your help @gregg-miskelly!

**Comment by ChristoWolf** [comment]
To anyone coming across this in the future:
The issue has been fixed in the [v2.23.2 pre-release](https://github.com/dotnet/vscode-csharp/releases/tag/v2.23.2).
